src/master-playlist-controller.js

```diff
@@ -370,7 +336,7 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     // Pause any alternative audio
     if (this.audioPlaylistLoader_) {
       this.audioPlaylistLoader_.pause();
-      this.audioPlaylistLoader_ = null;
+      this.audioPlaylistLoader_.dispose();
```

Comment:
yeah your right not sure what I was thinking. Will be committing an actual fix in a minute or two